bundles/org.openhab.binding.sunsynk/README.md

```diff
@@ -75,19 +90,30 @@ The SunSynk Inverter has the following channels.
 
 | Channel                        | Type                    | R/W | Description                       | Advanced |
 |--------------------------------|-------------------------|-----|-----------------------------------|----------|
-|battery-soc                     |Number:Dimensionless     | R   | Inverter battery % charge         | no       |
+|battery-soc                     |Number:Dimensionless     | R   | Inverter battery % charged        | no       |
 |battery-grid-voltage            |Number:ElectricPotential | R   | Battery dc electric-voltage       | no       |
 |battery-grid-current            |Number:ElectricCurrent   | R   | Battery dc electric-current       | no       |
 |battery-grid-power              |Number:Power             | R   | Battery dc electric-power         | no       |
 |battery-temperature             |Number:Temperature       | R   | Battery temperature               | no       |
 |inverter-ac-temperature         |Number:Temperature       | R   | Inverter ac temperature           | no       |
 |inverter-dc-temperature         |Number:Temperature       | R   | Inverter dc temperature           | no       |
-|inverter-grid-power             |Number:Power             | R   | Inverter ac electric-power        | no       |
-|inverter-grid-voltage           |Number:ElectricPotential | R   | Inverter ac electric-voltage      | no       |
-|inverter-grid-current           |Number:ElectricCurrent   | R   | Inverter ac electric-current      | no       |
-|inverter-solar-energy-today     |Number:Energy            | R   | Solar dc energy generated today   | no       |
-|inverter-solar-energy-total     |Number:Energy            | R   | Solar dc energy generated to date | no       |
-|inverter-solar-power-now        |Number:Power             | R   | Solar dc electric-current         | no       |
+|inverter-grid-power             |Number:Power             | R   | Grid ac electric-power            | no       |
+|inverter-grid-voltage           |Number:ElectricPotential | R   | Grid ac electric-voltage          | no       |
+|inverter-grid-current           |Number:ElectricCurrent   | R   | Grid ac electric-current          | no       |
+|inverter-grid-frequency         |Number:Frequency         | R   | Grid frequency                    | no       |
+|inverter-capacity               |Number:Power             | R   | Inverter energy capacity          | no       |
```

Comment:
Capacity is usually expressed in kWh as in Number:Energie. Or is this the maximum power the inverter can deliver?

When changed, it also needs to be adapted on other places (this readme item examples)